cxx-qt-build/src/lib.rs

```diff
@@ -85,6 +85,14 @@ enum ExtractedModule {
     None,
 }
 
+fn manifest_dir() -> String {
```

Comment:
maybe add a comment for this method describing why it's needed

Reply:
done in #108